crates/bevy_ecs/src/schedule/run_criteria.rs

```diff
@@ -407,7 +406,7 @@ impl Default for RunOnce {
     fn default() -> Self {
         Self {
             ran: false,
-            system_id: SystemId::new(),
+
```

Comment:
Unnecessary whitespace, I'm surprised rustfmt doesn't check for that

Reply:
Thanks!